pyClient/commands/constants.py

```diff
@@ -20,3 +20,5 @@
 
 WALLET_DIR_DEFAULT = "./notes"
 WALLET_USERNAME = "zeth"
+
+MERKLE_TREE_FILE_DEFAULT = "./merkle_tree"
```

Comment:
Let's be consistent with the naming of the constants:
In the same file you have:
`INSTANCEFILE_DEFAULT = "zeth-instance.json"`, `ADDRESSFILE_DEFAULT = "zeth-address.json"` (`FILE` is concatenated directly without `_`), and now you declare `MERKLE_TREE_FILE_DEFAULT = "./merkle_tree"`, with a `_FILE`.

Let's stick to `_FILE` everywhere.

Reply:
Also, let's use file extensions in order to inform on how the data is stored and should be read (`.dat` can be used for generic data files)

Reply:
Chaged to `_FILE_` everywhere, and used `.dat` for merkle tree data.